packages/gust/tsdx.config.js

```diff
@@ -10,7 +10,8 @@ module.exports = {
   rollup(config, options) {
     config.plugins.push(
       postcss({
-        modules: true,
+        // TODO: explore using css modules locally
```

Comment:
We aren't going to compile any tailwind css since we want to use variables at the project level